tests/test-rest-posts-controller.php

```diff
@@ -712,6 +712,18 @@ public function test_create_post_with_invalid_date_gmt() {
 		$this->assertErrorResponse( 'rest_invalid_param', $response, 400 );
 	}
 
+	public function test_create_post_with_quotes_in_title() {
+		wp_set_current_user( $this->editor_id );
+
+		$request = new WP_REST_Request( 'POST', '/wp/v2/posts' );
+		$params = $this->set_post_data( array(
+			'title' => "Rob O'Rourke's Diary",
+		) );
+		$request->set_body_params( $params );
+		$response = $this->server->dispatch( $request );
+		$this->check_create_post_response( $response );
```

Comment:
Can we have an explicit test for the response title, to make this test more precise?